htpy/__init__.py

```diff
@@ -344,6 +347,21 @@ def __repr__(self) -> str:
         return f"<{self.__class__.__name__} '<{self._name}{self._attrs}>'>"
 
 
+class Fragment:
+    """A collection of nodes without a wrapping element.
+
+    >>> content = Fragment("Hello ", None, i["world!"])
+    >>> print(content)
+    Hello <i>world!</i>
+    """
+
+    def __init__(self, *nodes: Node) -> None:
+        self.nodes = nodes
```

Comment:
```suggestion
        self._nodes = nodes
```

to avoid people from accessing it directly